feeds/trilliumtransit.com.dmfr.json

```diff
@@ -3158,6 +3223,20 @@
         "static_current": "https://data.trilliumtransit.com/gtfs/cctbus-mn-us/cctbus-mn-us.zip"
       }
     },
+    {
+      "id": "f-chapelhilltransit~nc~us",
```

Comment:
no need for a new feed record for this feed. we already have https://www.transit.land/feeds/f-dnru-chapel~hill~transit~nc~us

@westontrillium do you know of any meaningful difference between http://mychtransit.org/gtfs and https://data.trilliumtransit.com/gtfs/chapel-hill-transit-nc-us/chapel-hill-transit-nc-us.zip ?

Reply:
It's possible Trillium's data is being passed through to their CAD/AVL vendor which then publishes their own version for Google Maps and the like. Since Trillium's version is not the feed being ingested by consumers, I think it's safe to disregard it.